src/examples/20-seamless.html

```diff
@@ -18,10 +18,10 @@
             The seamless skin uses the height of your textarea as the minimum
             editor height.
         -->
-        <textarea class="wymeditor" style="min-height: 600px"></textarea>
+        <textarea class="wymeditor" style="min-height: 300px"></textarea>
         <input type="submit" class="wymupdate" />
     </form>
-    <div class="more-content">The toolbar will stop floating here.<div>
+    <p style="height: 5000px">The floating toolbar will stop here.</p>
```

Comment:
This does not go in the editor's document.

This is for something else.

The seamless skin has a "floating toolbar". It means that as you scroll down past it, it sticks to the top of the window, or something. And It doesn't get past the document view.

To be able to scroll down, there usually needs to be some content, below the editor. And this is what this `p` is for. That is why it has height.